src/template/mapping.json

```diff
@@ -474,5 +474,6 @@
   "bracket-error": 60390,
   "lock-small": 60391,
   "azure-devops": 60392,
-  "verified-filled": 60393
+  "verified-filled": 60393,
+  "newline": 61697
```

Comment:
Is there a reason why this doesn't follow the sequence from before? Should be `60394`

Reply:
I have copied this value from auto generated .css file (`npm run fonts`), converted hexadecimal code. Are all the codes manually written? It is enough to put just next number `60394`? Why they are not generating automaticaly? I think it will be more easy to add/remove icons, if mappig will be made in some different way to avoid using codes, and just to make a list of names with aliases.

Reply:
The auto-generated css files & unicode characters are generated based on the `mapping.json` file. The reason why we have to add them here is because we have a lot of references to the same icon using different aliases. Secondly, we use the icon font in our design tooling so if there are no codepoints they are auto-generated and can shift whenever there are new icons. This causes issues as the glyphs would always change whenever there are new ones. So we always want to assign a codepoint and follow the sequence from before.

Reply:
Updated `mapping.json` icon code.